.github/workflows/charterafrica-build-deploy.yml

```diff
@@ -7,13 +7,16 @@ env:
   DOKKU_REMOTE_BRANCH: "master"
   DOKKU_REMOTE_URL: "ssh://dokku@ui-1.prod.codeforafrica.org/charterafrica"
```

Comment:
@kilemensi I believe we can set the hostname to be consistent with what we have on Cloudflare, so this would be `charterafrica.prod`

Reply:
This workflow is to deploy to **dev** @thepsalmist. Still thinking about **prod** deployments

Reply:
Cool @kilemensi  LGTM

Reply:
Any ideas on how we should handle deployments to **dev** vs **prod** @thepsalmist ?

Reply:
Cool @thepsalmist ... For the now, can you check this setup for CI/CD

1. **DEV**: Every time a push is done to main (there is a path filter to make sure the changes were indeed made in an app).
2. **PROD**: The previous workflows where we only deploy if there is a version bump.

Reply:
Makes sense @kilemensi, although if i remember we introduced changesets to manage the version bump

Reply:
Yes, that hasn’t changed @thepsalmist: Youl’ll still use changesets to bump version in package.json and release workflow will bump the one in contrib/dokku and then a prod deployment will be made once the bump PR is merged to main.